test/realtime/tenants/batch_broadcast_test.exs

```diff
@@ -0,0 +1,530 @@
+defmodule Realtime.Tenants.BatchBroadcastTest do
```

Comment:
Should we add tests to batch controller, BroadcastHandler and ReplicationConnection to specify how the systems will react when payload size is reached?

* Should BroadcastHandler return an error when `ack: true` ? 
* Should ReplicationConnection log something? 
* And Batch Controller do we return any error?